ethereum/spec/src/main/java/tech/pegasys/teku/spec/SpecFactory.java

```diff
@@ -27,17 +31,30 @@ public static Spec createMinimal() {
     return create(Eth2Network.MINIMAL);
   }
 
+  public static Spec createMinimalAltair() {
```

Comment:
Probably a good idea to move this to a test fixture utility.  Since we don't really want to be loading explicit versions in production.

Reply:
Moved.